aws/src/integration/java/org/apache/iceberg/aws/dynamodb/TestDynamoDbCatalog.java

```diff
@@ -99,22 +100,23 @@ public void testCreateNamespace() {
                 .tableName(catalogTableName)
                 .key(DynamoDbCatalog.namespacePrimaryKey(namespace))
                 .build());
-    Assert.assertTrue("namespace must exist", response.hasItem());
-    Assert.assertEquals(
-        "namespace must be stored in DynamoDB",
-        namespace.toString(),
-        response.item().get("namespace").s());
-    Assertions.assertThatThrownBy(() -> catalog.createNamespace(namespace))
+    assertThat(response.hasItem()).as("namespace must exist").isTrue();
+    assertThat(response.item())
+        .as("namespace must be stored in DynamoDB")
+        .hasEntrySatisfying(
+            "namespace",
+            attributeValue -> assertThat(attributeValue.s()).isEqualTo(namespace.toString()));
```

Comment:
for a single key match what about simple assertion like 
```
assertThat(response.item().get("namespace").s()).as("namespace must be stored in DynamoDB").isEqualTo(namespace.toString());
```
For multiple keys validation it looks good to use `hasEntrySatisfying`. WDYT ?

Reply:
Thanks for the suggestion. I think your suggestion seems to be fine, but my change also checks if `response.item` has `namespace` or not.  But if the value extraction by`get("namespace")`, it doesn't check in the test. So if this part is like your suggestion, it's better to write like:

```
assertThat(response.item()).containsKey("namespace")
assertThat(response.item().get("namespace").s())...
```

That's why I'm using `hasEntrySatisfying` here. If I'm wrong here, please correct me.

Reply:
Makes sense, I think `hasEntrySatisfying` sounds good as per the above reasons.